### PR TITLE
Fade Script update

### LINKED ISSUE #1283: F key mapped to fade script
We shouldn't have [special mapped keys](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/2acbb55496aa689541be74f5e4b4f9bfe49b249a/Assets/HoloToolkit/Utilities/Scripts/FadeScript.cs#L68) like this in production scripts. Maybe adding a debug switch to enable/disable when testing is appropriate.

### LINKED ISSUE #1389: Mixed Reality Camera Parent Fade Plane impacts perf on HoloLens
## Overview
MRCP Fade Plane is transparent instead of black, and adds a full frame render that impacts performance on HoloLens

## Unity Editor Version
Any

## Mixed Reality Toolkit Release Version
272d5fd10c0b836a2a7bcb021a655dcf2288d7e9

### PR DESCRIPTION
**Pull request by StephenHodgson**
Fixes #1283
Fixes #1389

- Renamed FadeScript to FadeManager because it's a singleton (meta file should auto update reference in all scenes in all projects).
- Removed the F key input mapping that was used for testing.
- Updated Mixed Reality Teleport debug logs.  We now asset the FadeManager is initialized.  The log wasn't fully reflecting the fact that we could have removed this script if we were on the HoloLens as well.

### CODE REVIEW

File: Assets/HoloToolkit/Utilities/Scripts/FadeScript.cs.meta [hunk 1]
```diff
@@ -1,8 +1,9 @@
 fileFormatVersion: 2
-guid: 794a5a07720daf4479efca14d8ba4297
-timeCreated: 1495751254
+guid: 3227eb50378702c4eba7e61f471ee7c7
```
Comment (keveleigh) [inline Assets/HoloToolkit/Utilities/Scripts/FadeScript.cs.meta]: Won't this break existing links and discoverability of the `obsolete` message?

Comment (StephenHodgson) [reply Assets/HoloToolkit/Utilities/Scripts/FadeScript.cs.meta]: No, it'll update correctly.  I tested this in my own project.

Comment (keveleigh) [reply Assets/HoloToolkit/Utilities/Scripts/FadeScript.cs.meta]: Oh, you moved the GUID to point to the new script. What's the point of the obsolete marker then, if any existing references will migrate?

Comment (StephenHodgson) [reply Assets/HoloToolkit/Utilities/Scripts/FadeScript.cs.meta]: Just in case someone was referencing the class in code.

**Comment by StephenHodgson** [comment]
@keveleigh added fix for HoloLens perf.

**Pull request merged**